core/src/test/scala/astraea/spark/rasterframes/RasterFunctionsSpec.scala

```diff
@@ -0,0 +1,25 @@
+package astraea.spark.rasterframes
+
+import astraea.spark.rasterframes.TestData.randomTile
+import geotrellis.raster._
+import geotrellis.spark._
+import org.apache.spark.sql.functions._
+
+class RasterFunctionsSpec extends TestEnvironment with TestData {
```

Comment:
This should probably live in  `TileStatsSpec`.

https://github.com/locationtech/rasterframes/blob/develop/core/src/test/scala/astraea/spark/rasterframes/TileStatsSpec.scala

Reply:
Or `GTSQLSpec`

https://github.com/locationtech/rasterframes/blob/develop/core/src/test/scala/astraea/spark/rasterframes/GTSQLSpec.scala